internal/app/recordtester/recordtester_app.go

```diff
@@ -389,6 +420,31 @@ func (rt *recordTester) isCancelled() error {
 	return nil
 }
 
+func (rt *recordTester) signJwt(stream *api.Stream) (string, error) {
+	expiration := time.Now().Add(time.Minute * 5).Unix()
+	unsignedToken := jwt.NewWithClaims(jwt.SigningMethodES256, jwt.MapClaims{
+		"sub": stream.PlaybackID,
+		"pub": rt.publicKey,
+		"exp": expiration,
+	})
+
+	decodedPrivateKey, _ := base64.StdEncoding.DecodeString(rt.signingKey)
+
+	pk, err := jwt.ParseECPrivateKeyFromPEM(decodedPrivateKey)
+
+	if err != nil {
+		glog.Errorf("Unable to parse provided signing key for access control signingKey=%s", rt.signingKey)
+	}
+
+	token, err := unsignedToken.SignedString(pk)
+
+	if err != nil {
+		glog.Errorf("Unable to sign JWT with provided private key for access control signingKey=%s", rt.signingKey)
+	}
```

Comment:
We need to return all these errors. Cannot just log.
```suggestion
	if err != nil {
		glog.Errorf("Unable to parse provided signing key for access control signingKey=%s", rt.signingKey)
		return "", err
	}

	token, err := unsignedToken.SignedString(pk)

	if err != nil {
		glog.Errorf("Unable to sign JWT with provided private key for access control signingKey=%s", rt.signingKey)
		return "", err
	}
```